manifests/m/Microsoft/PIX/22.1.24.1/Microsoft.PIX.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.PIX
+PackageVersion: 22.1.24.1
+Installers:
+- Architecture: x64
+  InstallerType: burn
+  InstallerUrl: https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RWSCcX
+  InstallerSha256: B73B97D223B7DC0BDD16A8D2F5B8DDC1BC7AE855EC420C028FAE3E6B4BCF636F

```

```suggestion
  InstallerSha256: B73B97D223B7DC0BDD16A8D2F5B8DDC1BC7AE855EC420C028FAE3E6B4BCF636F
  Scope: machine
```
`C:\Program Files\Microsoft PIX\2201.24`